docs/app/pages/components/sections/date-time-picker/time-picker/snippets/app.html

```diff
@@ -10,67 +10,42 @@
             <accordion-group heading="Customize Example...">
                 <div class="row uxd-customize-row">
                     <div class="col-md-6 col-sm-6">
-                        <div>
-                            <label for="minHour">min hour</label>
-                            <ux-number-picker [(value)]="minHour" min="0" max="23" (valueChange)="minTime.setHours($event)"></ux-number-picker>
-                        </div>
+                        <label for="myMinTime">min time:</label>
```

Comment:
Sorry for the continuing confusion - this label text needs to match the name of the Bootstrap property:

````
<label for="myMinTime">min</label>
````

The same for all properties.